README.rst

```diff
@@ -71,7 +70,8 @@ Usage
       -m MESSAGE, --message MESSAGE
                             Specify a commit message (comment) for the new image.
       -c, --cleanup         Remove source image from Docker after squashing
-      --tmp-dir TMP_DIR     Temporary directory to be created and used
+      --tmp-dir TMP_DIR     Temporary directory to be created and used. This implies development mode and will NOT clean
```

Comment:
I would not even mention development mode anymore. This is now an internal thing IMHO. I think this message can confuse people. I would maybe even extend what the temporary dir is used for.